README.md

```diff
@@ -23,9 +23,9 @@ Ensure you have service-manager python environment setup:
 
 `source ../servicemanager/bin/activate`
 
-`sm --start DC_TWSM_ALL`
+`sm2 --start DC_TWSM_ALL`
```

Comment:
Can this use the same profile as stop? Meaning - can you use SECURE_MESSAGE profile to start and stop? Might save some time over starting all?